bootstrap-demo/kubernetes-dns/test.sh

```diff
@@ -20,23 +21,20 @@ then
   exit -1
 fi
 
-POD=$(kubectl get pods | grep demo-kubernetes-dns | grep Running | head -n1 | awk '{ print $1 }')
+POD=$(kubectl get pods --namespace akka-bootstrap | grep akka-bootstrap-demo | grep Running | head -n1 | awk '{ print $1 }')
 
 for i in {1..10}
 do
   echo "Checking for MemberUp logging..."
-  kubectl logs $POD bootstrap-demo-kubernetes-dns | grep MemberUp || true
-  [ `kubectl logs $POD bootstrap-demo-kubernetes-dns | grep MemberUp | wc -l` -eq 3 ] && break
+  kubectl logs $POD --namespace akka-bootstrap | grep MemberUp || true
+  [ `kubectl logs --namespace akka-bootstrap $POD | grep MemberUp | wc -l` -eq 3 ] && break
   sleep 3
 done
 
 if [ $i -eq 10 ]
 then
   echo "No 3 MemberUp log events found"
-  for POD in $(kubectl get pods | grep demo-kubernetes-dns | grep Running | awk '{ print $1 }')
-  do
-    echo "Logging for $POD"
-    kubectl logs $POD bootstrap-demo-kubernetes-dns
-  done
+  echo "=============================="
+  kubectl logs $POD
```

Comment:
I think in this case it is actually useful to collect the logging from all 3 pods, not just the first one, to diagnose what went wrong

Reply:
shame on me for treating `POD` as a `var` though, I guess :D